.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: python
 python:
-  - "2.6"
   - "2.7"
 install:
-  - pip install pep8
+  - pip install pylama
```

Comment:
Nice one! I haven't known about this tool. It looks great that it wraps a couple of tools into a single one.